sdk/storage/azure-storage-blobs/test/ut/blob_container_client_test.cpp

```diff
@@ -227,6 +227,62 @@ namespace Azure { namespace Storage { namespace Test {
     EXPECT_TRUE(std::includes(listBlobs.begin(), listBlobs.end(), p1Blobs.begin(), p1Blobs.end()));
   }
 
+  TEST_F(BlobContainerClientTest, ListBlobsFlatPageable)
```

Comment:
Can we add a test for hierarchical and not just the flat API?

Reply:
Yes, I just want this example get approved first before I start working on the others.